### DISCUSSION TITLE
TooManyRequests: Requests limit reached, try increasing throttling or wait before trying again

### DISCUSSION THREAD

**Discussion by ghost** [sergioteula/python-amazon-paapi#125] [Help]
Hello,

I was using the API for the first time 12 hours ago. I downloaded the python sdk and inserted the access / secret key and the correct informations about my current marketplace (brazil). I was getting this error: 

Error calling PA-API 5.0!
Status code: 401
Errors : {"__type":"com.amazon.paapi5#UnrecognizedClientException","Errors":[{"Code":"UnrecognizedClient","Message":"The Access Key ID or security token included in the request is invalid."}]}

I doubled checked so many times my credentials, recreated it and decided to use this sample code posted here:

from amazon_paapi import AmazonApi
KEY = "XXXXXX"
SECRET = "XXXXXX"
TAG = "XXXXXX-20"
COUNTRY = "BR"
amazon = AmazonApi(KEY, SECRET, TAG, COUNTRY)
item = amazon.get_items('B0BFTFK1BN')[0]
print(item.item_info.title.display_value)

Now i am getting this:
amazon_paapi.errors.exceptions.TooManyRequests: Requests limit reached, try increasing throttling or wait before trying again

It's been taking like 6 hours.

How much time should i wait to keep trying?

Thanks

**Comment by al3xam** [comment]
Hi, 
I have the same problem, although I have never managed to make it work, it always gives me back

Request limit reached, try increasing the throttle or wait before trying again.
Have you managed to fix it?
